src/main.rs

```diff
@@ -148,7 +151,11 @@ where
                         on_winch({
                             Box::new(move || {
                                 let mut ui = ui.lock().unwrap();
-                                ui.draw(paused.load(Ordering::SeqCst), dns_shown);
+                                ui.draw(
+                                    paused.load(Ordering::SeqCst),
+                                    dns_shown,
+                                    std::time::Duration::new(131, 0),
```

Comment:
I think we should probably replace this with the actual duration :)

Reply:
Yeah, figuring out why when resizing the window the timer changes to 02:11 were a confusing 2 minutes. ;P

Reply:
Opps... Sorry! 😅